ush/mesoscale/mesoscale_check_input_data.py

```diff
@@ -302,13 +302,13 @@ def fname_constructor(template_str, IDATE="YYYYmmdd", IHOUR="HH",
             missing_fcst_files.append(os.path.split(fcst_path)[-1])
             missing_fcst_paths.append(fcst_path)
     
-    # Print error and send an email for missing data
+    # Print error (mdf) and send an email for missing data
     if missing_fcst_paths:
         print(f"ERROR: The following forecasts were not found:")
```

Comment:
@RoshanShrestha-NOAA Can we please change these print statements to say "WARNING" instead of "ERROR"? The word "ERROR" should only be printed if the code fails. For example from the NCO EE2 checklist:

Are fatal errors prefaced with ‘FATAL ERROR’ and non-fatal/warning errors prefaced with ‘WARNING’? 

CCing @MarcelCaron-NOAA in case this is in CAM and should be addressed in the next PR.

Reply:
Hi @AliciaBentley-NOAA. Done all the suggested changes.